search.py

```diff
@@ -28,20 +38,36 @@ def main():
 
     tokenizer = DistilBertTokenizerFast.from_pretrained('distilbert-base-uncased')
 
-    timestamp = datetime.now().strftime("%m-%d-%Y %I:%M:%S.%f %p")
+    scheduler = ASHAScheduler(
+        metric="F1",
+        mode="max",
+        max_t=100, # number of eval_every batches
+        grace_period=10,
```

Comment:
This seems fine, but an explanation here is probably a good idea because this param is so important

Reply:
I made some changes to this section, do you want to take a second look?